internal/bbr/readme.go

```diff
@@ -0,0 +1,5 @@
+// package bbr implements the BBR congestion control algorithm, as
+// described in:
+//
+// https://queue.acm.org/detail.cfm?id=3022184
+package bbr
```

Comment:
Maybe rename the file to the standard `doc.go`?

Reply:
Done.